appcelerator/titanium_mobile
# fix(ios): TableView "sectionCount" property crash

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-13903

**Unit Test Note:**
RecylerView PR #12029 is making heavy changes to the `ti.ui.tableview.test.js` script. In order to avoid a huge amount of conflicts, we should re-enable all `iosBroken` unit tests via that PR instead of this one.
_(I've already verified this PR's change fixes the unit tests.)_

**Test:**
1. Build and run the below on iOS.
2. Verify an alert displays "sectionCount: 2" on app startup.

```javascript
var section1 = Ti.UI.createTableViewSection({ headerTitle: "Colors" });
section1.add(Ti.UI.createTableViewRow({ title: "Red" }));
section1.add(Ti.UI.createTableViewRow({ title: "Green" }));
section1.add(Ti.UI.createTableViewRow({ title: "Blue" }));

var section2 = Ti.UI.createTableViewSection({ headerTitle: "Fruit" });
section2.add(Ti.UI.createTableViewRow({ title: "Apples" }));
section2.add(Ti.UI.createTableViewRow({ title: "Bananas" }));
section2.add(Ti.UI.createTableViewRow({ title: "Peaches" }));

var window = Ti.UI.createWindow();
var tableView = Ti.UI.createTableView({
	data: [section1, section2],
});
window.add(tableView);
window.addEventListener("open", function() {
	alert("sectionCount: " + tableView.sectionCount);
});
window.open();
```
